docs/guides/integration/aws-lambda.md

```diff
@@ -0,0 +1,407 @@
+# Using uv with AWS Lambda
+
+[AWS Lambda](https://aws.amazon.com/lambda/) is a serverless computing service that lets you run
+code without provisioning or managing servers.
+
+You can use uv with AWS Lambda to manage your Python dependencies, build your deployment package,
+and deploy your Lambda functions.
+
+!!! tip
+
+    Check out the [`uv-aws-lambda-example`](https://github.com/astral-sh/uv-aws-lambda-example) project for
+    an example of best practices when using uv to deploy an application to AWS Lambda.
+
+## Getting started
+
+To start, assume we have a minimal FastAPI application with the following structure:
+
+```plaintext
+project
+├── pyproject.toml
+└── app
+    ├── __init__.py
+    └── main.py
+```
+
+Where the `pyproject.toml` contains:
+
+```toml title="pyproject.toml"
+[project]
+name = "uv-aws-lambda-example"
+version = "0.1.0"
+requires-python = ">=3.13"
+dependencies = [
+    # FastAPI is a modern web framework for building APIs with Python.
+    "fastapi",
+    # Mangum is a library that adapts ASGI applications to AWS Lambda and API Gateway.
```

Comment:
As a minor note, you could move this out and use a `!!! note` admonition instead so you can link to the library docs. Don't feel strongly though.